src/server/scripts/Kalimdor/zone_winterspring.cpp

```diff
@@ -40,48 +39,6 @@ EndContentData */
 #include "TemporarySummon.h"
```

Comment:
check if gossip includes is unneed after this changes :)

Reply:
I thought it was because go_elune_fire uses GossipHello but I now see that that's not in ScriptedGossip.h, you're right. Thanks for the heads up.